src/api/firebase.js

```diff
@@ -11,6 +28,7 @@ import { getFutureDate } from '../utils';
  * @see https://firebase.google.com/docs/firestore/query-data/listen
  */
 export function streamListItems(listId, handleSuccess) {
+	// TODO: check if the token is a valid token in firebase
```

Comment:
Is this todo something that you'd like to complete in this PR or is that what `doesCollectionExist` is solving above?

Reply:
It is covered in the doesCollectionExist, comment just stuck around. Cleaned up in my recent commit. Thanks!